.gitlab-ci.yml

```diff
@@ -0,0 +1,56 @@
+stages:
+    - build
+    - deploy
+
+.maven-build:
```

Comment:
Pipeline job. The leading dot means the job will be ignored and not executed in the pipeline. This is useful for inheritance, as this can be used as a parent "abstract" job.

Reply:
```suggestion
# Not executed, parent job definition for Maven builds
.maven-build:
```